src/Attachment.php

```diff
@@ -1,35 +1,66 @@
 <?php
 
+/*
+ * (c) Henrik Bjornskov <henrik@bjrnskov.dk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace Stampie;
 
+/**
+ * @package Stampie
+ */
 final class Attachment
 {
+    /**
+     * @var string
+     */
     private $path;
+
+    /**
+     * @var string
+     */
     private $name;
 
+    /**
+     * @param string $path
+     * @param string $name
+     */
     public function __construct($path, $name = null)
     {
         $this->path = $path;
         $this->name = $name;
     }
 
+    /**
+     * @return string
+     */
     public function getName()
     {
         return $this->name ?: basename($this->path);
     }
 
+    /**
+     * @return string
+     */
     public function getContentType()
     {
-        $file = new \finfo(FILEINFO_MIME_TYPE);
-
-        return $file->file($this->path);
+        return (new \finfo(FILEINFO_MIME_TYPE))->file($this->path);
```

Comment:
please don't remove the support for PHP 5.3 here (the composer.json says 5.3.3 as min requirement)